CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ PrestaShop 1.7 is written mostly in PHP. Other languages used throughout are Jav
 To contribute to the project, you should ideally be familiar with Git, the source code management system that PrestaShop uses, with the official repository being hosted on Github: 
 * You can learn more about Git here: https://try.github.io/ (there are many tutorials available on the Web).
 * You can get help on Github here: https://help.github.com/.
-* Windows users can get a nice interface for Git by installing TortoiseGit: https://tortoisegit.org/
+* Windows users can get a nice interface for Git by installing TortoiseGit: https://tortoisegit.org/.
```

Comment:
I think it's intended to ease copy-paste into browser bar.

We can improve this into a clickable href:
```suggestion
* Windows users can get a nice interface for Git by installing [TortoiseGit](https://tortoisegit.org/).
```

Reply:
It can also be done for https://try.github.io/ and https://help.github.com/.